test/helpers/gameflowwrapper.js

```diff
@@ -73,12 +73,6 @@ class GameFlowWrapper {
         this.guardCurrentPhase('dominance');
     }
 
-    completeTaxationPhase() {
-        this.guardCurrentPhase('taxation');
-        // TODO: Discard down to reserve in case of tests that fill up the player's hand
-        this.eachPlayerInFirstPlayerOrder(player => player.clickPrompt('End Round'));
```

Comment:
This method should stay for completeness (there's one for every other phase), but is should call skipActionWindow instead of clicking 'End Round'. Even if the default is now no pause, this method should be available in tests.

Reply:
The phase has changed before it gets to the method so the assertion fails, I don't see how I can readd it without adding some artificial step or turning on the window just for tests